oe_contact_forms.module

```diff
@@ -6,3 +6,327 @@
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+  $form['privacy_policy'] = [
+    '#type' => 'textarea',
+    '#title' => t('Privacy policy'),
+    '#description' => t("Please specify a privacy policy message to display on the form."),
+    '#states' => [
+      'required' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+      'visible' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+    ],
+    '#default_value' => $contact_form->getThirdPartySetting('oe_contact_forms', 'privacy_policy', ''),
+  ];
+  $form['includes_fields_in_auto_reply'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Includes fields in auto-reply'),
+    '#description' => t("Check this box if you would like to include in the auto-reply also the values submitted by the user."),
+    '#default_value' => $contact_form->getThirdPartySetting('oe_contact_forms', 'includes_fields_in_auto_reply', FALSE),
+    '#states' => [
+      'visible' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+        ':input[name="reply[value]"]' => ['filled' => TRUE],
+      ],
+    ],
+  ];
+  $form['allow_canonical_url'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Allow canonical URL'),
+    '#description' => t("Check this box if you would like this contact form to be exposed at its default canonical URL."),
+    '#default_value' => $contact_form->getThirdPartySetting('oe_contact_forms', 'allow_canonical_url', FALSE),
+    '#states' => [
+      'visible' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+  $form['expose_as_block'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Expose as block'),
+    '#description' => t("Check this box if you would like to expose this contact form as a block."),
+    '#default_value' => $contact_form->getThirdPartySetting('oe_contact_forms', 'expose_as_block', TRUE),
+    '#states' => [
+      'visible' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+  $form['optional_fields'] = [
+    '#type' => 'checkboxes',
+    '#title' => t('Optional fields'),
+    '#options' => [
+      'oe_country_residence' => t('Country of residence'),
+      'oe_telephone' => t('Phone'),
+    ],
+    '#description' => t("Please specify which optional fields you'd like to include in the form."),
+    '#default_value' => $contact_form->getThirdPartySetting('oe_contact_forms', 'optional_fields', []),
+    '#states' => [
+      'visible' => [
+        ':input[name="is_corporate_form"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+
+  $form['#entity_builders'][] = '_oe_contact_forms_contact_form_builder';
+  $form['#validate'][] = '_oe_contact_forms_email_validate';
+}
+
+/**
+ * Default ajax callback for Topics.
+ *
+ * @param array $form
+ *   The form.
+ * @param Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state.
```

Comment:
Missing return value in the doc. Grump didnt raise this issue?

Reply:
As I see it's a void function so actually there is no return.

Reply:
it was array, it is fixed